.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Data Sources]or[Resources]:[ibm_pi_feature]"
```

Comment:
Data Sources or Resources do not make sense here.

Reply:
@yussufsh Removed it and changed file format